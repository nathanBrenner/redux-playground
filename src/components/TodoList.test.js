import React from "react";
import TodoList from "./TodoList";
import { shallow } from "enzyme";
import renderer from "react-test-renderer";

describe("TodoList", () => {
  test("renders without crashing", () => {
    const todos = [{ id: "1", completed: false, text: "do something" }];
    const onTodoClick = () => true;
    const props = { todos, onTodoClick };

    shallow(<TodoList {...props} />);
  });

  test("snapshot", () => {
    const todos = [{ id: "1", completed: false, text: "do something" }];
    const onTodoClick = () => true;
    const props = { todos, onTodoClick };
    const component = renderer.create(<TodoList {...props} />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
