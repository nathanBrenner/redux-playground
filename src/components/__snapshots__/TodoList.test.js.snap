// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TodoList snapshot 1`] = `
<ul>
  <li
    onClick={[Function]}
    style={
      Object {
        "textDecoration": "none",
      }
    }
  >
    do something
  </li>
</ul>
`;
